src/main/resources/application.yml

```
@@ -17,6 +17,17 @@ spring:
       fail-on-empty-beans: false
   main:
     allow-bean-definition-overriding: true
+  flyway:
+    enabled: true
+  datasource:
+    url: jdbc:postgresql://${DB_HOST}:${DB_PORT}/familie-ef-iverksett
```

Tror `DB_NAME` skal fungere her
```suggestion
    url: jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}
```